editor/src/clj/editor/code/data.clj

```diff
@@ -2534,7 +2541,15 @@
   (assert (breakpoint-region? region))
   (.row (cursor-range-start region)))
 
-(defn toggle-breakpoint [lines regions rows]
+(defn region->breakpoint [resource region]
+  (let [condition (:condition region)]
+    (cond-> {:resource resource
+             :row (breakpoint-row region)
+             :enabled (:enabled region)}
+      condition
+      (assoc :condition condition))))
```

Comment:
What's the reason for using a separate data structure for breakpoints instead of using regions?